app/src/main/java/ru/ifmo/md/lesson8/MainActivity.java

```diff
@@ -0,0 +1,108 @@
+package ru.ifmo.md.lesson8;
+
+import android.app.ActionBar;
+import android.app.Activity;
+import android.app.FragmentManager;
+import android.content.ContentValues;
+import android.os.Bundle;
+import android.support.v4.widget.DrawerLayout;
+import android.view.Menu;
+import android.view.MenuItem;
+
+
+public class MainActivity extends Activity
+        implements NavigationDrawerFragment.NavigationDrawerCallbacks, WeatherFragment.OnFragmentInteractionListener, CitySearchFragment.OnFragmentInteractionListener {
+
+    private NavigationDrawerFragment mNavigationDrawerFragment;
+
+    static long lastUpdate = 0;
```

Comment:
This is not stored in database because there is no sense in storing it in database. It is set shortly after app launch - that is, as soon as weather is downloaded. It should not be stored between app launches, so storing it in static variable makes much more sense than storing it in database.